magstock/__init__.py

```diff
@@ -0,0 +1,75 @@
+from uber.common import *
+from magstock._version import __version__
+
```

Comment:
Side note: there are two major things which we are not yet doing in this repo (but will before MAGStock happens) which we expect event-specific plugins to commonly do:

1) Configure event-specific emails.

2) Add new sections to the website.

I'll cover those in the pull requests when those things are implemented.  (Though the emails part is just the same as how we do emails in Uber now.)